src/goSuggest.ts

```diff
@@ -53,7 +53,15 @@ export class GoCompletionItemProvider implements vscode.CompletionItemProvider {
 				let autocompleteUnimportedPackages = config['autocompleteUnimportedPackages'] === true && !lineText.match(/^(\s)*(import|package)(\s)+/);
 
 				if (lineText.match(/^\s*\/\//)) {
-					return resolve([]);
+					let nextLine = document.lineAt(position.line + 1).text;
+					let memberName = nextLine.replace(/\s+/g, ' ').trim().split(' ');
+					if (!nextLine.startsWith('\t')) {
```

Comment:
I am guessing your assumption here is that the file is always formatted, which might not be the case. Instead look for `func`, `var` and `type` declarations in the `nextLine`.

Reply:
Will take these considerations into account and modify code.